src/frontend/components/Chat/SharedChatComponents/ChatMessageMetadatas/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { Box, Text } from 'grommet';
+import { DateTime } from 'luxon';
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+import styled from 'styled-components';
+import { theme } from '../../../../utils/theme/theme';
+import { ChatAvatar } from '../ChatAvatar';
+
+interface ChatMessageMetadatasProps {
+  msgDatetime: DateTime;
+  msgSender: string;
+}
+
+const messages = defineMessages({
+  senderTitle: {
+    defaultMessage: 'The name of the sender of the message',
```

Comment:
```suggestion
    defaultMessage: 'The name of the message's sender',
```

Reply:
done in [058447a951165114aa40a1a78e5a26b7ff907df4](https://github.com/openfun/marsha/pull/1247/commits/058447a951165114aa40a1a78e5a26b7ff907df4)